examples/charts/authenticated-realm-web/README.md

```diff
@@ -10,16 +10,16 @@ MongoDB Charts allows you to create visualizations of your MongoDB data using a
 
```

Comment:
I think the link on line 7 be changed to `...realm-web` too

Reply:
good catch! You're a life saver 😄